source/edge-server/manage-edge-servers.txt

```diff
@@ -0,0 +1,518 @@
+
+         .. step:: Call the Create Edge Server Endpoint
+
+            Call the <CREATE EDGE SERVER> endpoint to add an Edge Server 
+            instance to your service. You must provide the the App's internal 
+            ``appId`` hex string:
+
+            .. code-block:: shell
+
+               INSERT API DETAILS HERE
+
+            This returns <INSERT RETURN DETAILS HERE>.
+
+         .. step:: Install the Edge Server code
+
+            Copy and paste the installation script into the terminal on the
+            device where you want to install the Edge Server code. For more 
+            details, refer to :ref:`edge-server-install-and-configure`.
+
+.. _list-edge-server-instances:
+
+List Edge Server Instances
+--------------------------
+
+You can get a list of your Edge Server instances in the Atlas UI,
+or from the Admin API.
+
+.. tabs-realm-admin-interfaces::
+
+   .. tab::
+      :tabid: ui
+
+      The Edge Server dashboard lists a brief summary for each Edge Server
+      instance, including the date and time of the last connection, and a 
+      snapshot of its status. You can optionally view details or logs for each 
+      instance.
+
+      To view the Edge Server dashboard, choose :guilabel:`Edge Server` in
+      the left navigation sidebar.
+
+      To view the details for a specific Edge Server instance, click 
+      :guilabel:`View Details` for that Edge Server instance. Alternately,
+      click the :guilabel:`App Logs` button to browse logs associated with 
+      the instance.
+
+   .. tab::
+      :tabid: api
+
+      .. procedure::
+
+         .. step:: Authenticate a MongoDB Atlas User
+
+            .. include:: /includes/api-authenticate-instructions.rst
+
+         .. step:: Get the List of Edge Server Instances
+
+            Call the <LIST EDGE SERVER> endpoint to get a list of all the 
+            Edge Servers instances:
+
+            .. code-block:: shell
+
+               INSERT API DETAILS HERE
+
+            Find the ID of the instance whose details you want to view.
+
+         .. step:: Call the Edge Server Info Endpoint
+
+            Call the <GET EDGE SERVER INFO> endpoint with the ID of the 
+            instance whose details you want to view.
+
+            .. code-block:: shell
+
+               INSERT API DETAILS HERE
+
+.. _edge-server-details:
+
+Edge Server Instance Details
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The information available in the Edge Server details varies depending on 
+whether the Edge Server has been configured and connected to Atlas, or whether
+configuration and connection are still pending.
+
+.. _edge-server-pending:
+
+Pending Edge Server Instances
+`````````````````````````````
+
+When an Edge Server instance has not yet connected to Atlas for the first time, 
+the instance's :guilabel:`Health` status is :guilabel:`Pending`.
+
+While an Edge Server instance is in :guilabel:`Pending` state, the detail
+view includes:
```

Comment:
are long bullet lists the the best way to communicate this? it's difficult to parse the two sub-sections and their content (are we revisiting this section once the UI is finalized?)